app/models/user.rb

```diff
@@ -2,4 +2,18 @@ class User < ApplicationRecord
   has_paper_trail
 
   belongs_to :organisation
+
+  def self.from_profile!(government_gateway_profile)
```

Comment:
What is the bang method signifying?

Reply:
Chatted about this offline - essentially signifies persistence/save!